### ISSUE TITLE
Wrong 7 days value change USD

### ISSUE THREAD

**Issue by eudora-fabia** [lorcalhost/BTB-manager-telegram#129]
Hi, I received this message

```
Last update: 06:18:45 01/08/2021

Current coin WIN:
 - Balance: 1300659.039 WIN
 - Exchange rate purchased: 0.0003565 USDT/WIN 
 - Exchange rate now: 0.0003685 USD/WIN
 - Change in value: 3.26 %
 - Value in USD: 479.29 USD
 - Value in BTC: 0.01300659 BTC

Bought for 464.15 USDT
1 day value change USD: 0 %
7 days value change USD: 16949003.69 %
```

Currently using commit c0f71c5

**Comment by shhhmel** [comment]
Similar to this one: https://github.com/lorcalhost/BTB-manager-telegram/issues/130

**Comment by titulebolide** [comment]
Might be fixed by #143

**Comment by titulebolide** [comment]
Closing this in favor of #130
